docs/install.rst

```diff
@@ -59,30 +59,30 @@ To manually turn ON/OFF the GPU CUDA compilation, see :ref:`these instructions <
  2. to make the compilation easier, let's work in a Python environment. |galario| works with both Python 2 and 3.
 
     For example, if you are using the `Anaconda <https://www.continuum.io/downloads>`_ distribution, you can create and
-    activate a Python 3 environment with:
+    activate a Python 3.6 environment with:
 
     .. code-block:: bash
 
-        conda create --name galario3 python=3 numpy cython pytest
+        conda create --name galario3 python=3.6 numpy cython pytest scipy
         source activate galario3
 
- 2. Use `cmake` to prepare the compilation and `make all` to compile. From within `galario/build/`:
+ 3. Use `cmake` to prepare the compilation from within `galario/build/`:
 
     .. code-block:: bash
 
-       cmake ..
+       cmake -DCMAKE_INSTALL_PREFIX=$CONDA_PREFIX ..
 
     This command will produce configuration and compilation logs listing all the libraries and the compilers that are being used.
     It will use the internet connection to automatically download `this <https://github.com/UCL/GreatCMakeCookOff>`_ additional library (1.5 MB).
+    The option `-DCMAKE_INSTALL_PREFIX` sets the directory where |galario| will be installed equal to the content of `$CONDA_PREFIX`. If you are inside a `conda` environment, then `$CONDA_PREFIX` is already populated with the correct directory. For |galario|v > 1.2 it is mandatory to provide this option.
```

Comment:
I think the new paragraph is not needed anymore.

Reply:
I agree

Reply:
```suggestion
```